akvo/rest/views/report.py

```diff
@@ -52,6 +52,45 @@ def report_formats(request):
     })
 
 
+@api_view(['GET'])
+def organisation_reports(request):
```

Comment:
I think we should add some tests for this new end-point, as well. I think we have some tests for the reports end-point that can be moved over or translated.